hack/antrea/cyclonus-job-github-action.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cyclonus
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - command:
+            - ./cyclonus
+            - generate
+            - --perturbation-wait-seconds=15
+            - --server-protocol=tcp,udp
+            - --include=conflict
```

Comment:
Could even reduce the number of tests further, to about 4 or so to make the job quicker.